assets/templates/kubernetes.pod/schema-b/gotext.tpl

```diff
@@ -12,5 +12,7 @@
ope-west1-d", "node_kubernetes_io/instance-type":"e2-standard-4", "beta_kubernetes_io/os":"linux", "cloud_google_com/gke-boot-disk":"pd-balanced", "kubernetes_io/os":"linux", "cloud_google_com/private-node":"false", "cloud_google_com/gke-logging-variant":"DEFAULT", "beta_kubernetes_io/arch":"amd64" } }, "pod":{ "uid": "demo-pod-{{ $rangeofid }}", "start_time": "{{$picktimedate}}T{{$timehour}}:{{ $resttime._1 }}:{{ $resttime._2 }}:{{ $resttime._3}}", "memory":{ "rss":{ "bytes":"{{generate "Bytes"}}" }, "major_page_faults":0, "usage":{ "node":{ "pct": {{divf $pctmem 100}} }, "bytes": "{{generate "Bytes"}}", "limit":{ "pct":{{divf $pctmem 100}} } }, "available":{ "bytes":0 }, "page_faults":1386, "working_set":{ "bytes": "{{generate "Bytes"}}", "limit":{ "pct": {{divf $pctmem 100}} } } }, "ip":"{{generate "Ip"}}", "name":"demo-pod-{{ $rangeofid }}", "cpu":{ "usage":{ "node":{ "pct":{{divf $pctcpu 100}} }, "nanocores": {{mul $nanocores 1000}}, "limit":{ "pct":{{divf $pctcpu 100}} } } }, "network":{ "tx":{ "bytes": {{ $txbytes }}, "errors":0 }, "rx":{ "bytes": {{ $rxbytes }}, "errors":0 } } }, "namespace":"demo-{{ $rangeofid }}", "namespace_uid":"demo-{{ $rangeofid }}", "replicaset":{ "name":"demo-deployment-{{ $rangeofid }}" }, "namespace_labels":{ "kubernetes_io/metadata_name":"demo-{{ $rangeofid }}" }, "labels":{ "app":"demo", "pod-template-hash":"{{ $rangeofid }}", "app-2":"demo-2", "app-1":"demo-1" }, "deployment":{ "name":"demo-deployment-{{ $rangeofid }}" } }, "cloud": { "provider": "gcp", "availability_zone": "europe-west1-d", "instance":{ "name": "{{ $agentName }}" , "id": "{{ $agentId }}" }, "machine":{ "type":"e2-standard-4" }, "service":{ "name":"GCE" }, "project":{ "id":"elastic-obs-integrations-dev" }, "account":{ "id":"elastic-obs-integrations-dev" } }, "orchestrator":{ "cluster":{ "name":"kubernetes-scale", "url":"https://{{generate "Ip"}}" } }, "service":{ "address": "https://{{ $agentName }}:10250/stats/summary", "type":"kubernetes" }, "data_stream":{ "namespace":"default", "type":"metrics", "dataset":"kubernetes.pod" }, "ecs": { "version": "8.2.0" }, "agent": { "id": "{{ $agentId}}", "name": "{{ $agentName }}" , "type": "metricbeat", "version": "{{ $agentVersion }}", "ephemeral_id": "{{ $agentEphemeralid }}" }, "elastic_agent": { "id": "{{ $agentId }}" , "version": "{{ $agentVersion }}", "snapshot": {{ generate "agent.snapshot" }} }, "metricset":{ "period": "{{ $period }}" , "name":"pod" }, "event":{ "duration": "{{generate "event.duration"}}", "agent_id_status": "verified", "ingested": "{{$picktimedate}}T{{$timehour}}:{{ $resttime._1 }}:{{ $resttime._2 }}:{{ $resttime._3}}", "module":"kubernetes", "dataset":"kubernetes.pod" }, "host":{ "hostname":"host-{{ $nodeid }}", "os":{ "kernel":"5.10.161+", "codename":"focal", "name":"Ubuntu", "type":"linux", "family":"debian", "version":"20.04.5 LTS (Focal Fossa)", "platform":"ubuntu" }, "containerized":false, "name": "host-{{ $nodeid }}", "id": "host-{{ $nodeid }}", "architecture":"x86_64"}}
```

Comment:
Seem comment above

Reply:
The generator tool does not produce the output inline and this was needed for rally (FYI linebreaks was making rally to fail). This is fixed in the elastic-package, the inline output.

But for this review I had included the [multiline versions of templates](https://github.com/elastic/elastic-integration-corpus-generator-tool/blob/117f5f724509d845a282acaae84d366ad855766a/assets/templates/kubernetes.pod/schema-b/gotext_multiline.tpl) just for review.

Reply:
> This is fixed in the elastic-package, the inline output.

Does this mean in the future we can have it expanded?

Reply:
Yes, see https://github.com/elastic/integrations/blob/main/packages/kubernetes/_dev/benchmark/rally/pod-benchmark/template.ndjson is expanded

Reply:
❤️